src/screens/homePageAuth/SignIn.js

```diff
@@ -27,7 +27,9 @@ export default function SignIn({ navigation }) {
     const sendRegisterData = async (data) =>{
         setLoading(true);
         try {
+            console.log(JSON.stringify(data))
             const result = await signup(data);
+            console.log(JSON.stringify(result))
```

Comment:
```suggestion
            const result = await signup(data);
```

No nos queremos llevar de console.log